network-protection/network-protection-impl/src/test/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementViewModelTest.kt

```diff
@@ -197,6 +197,7 @@ class NetworkProtectionManagementViewModelTest {
 
     @Test
     fun whenOnNetpToggleClickedToDisabledThenUnregisterFeature() = runTest {
+        vpnRemoteFeatures.showExcludeAppPrompt().setEnabled(Toggle.State(enable = false))
```

Comment:
@aitorvs is this not correct way of setting the feature state in test? I just see that it is being flagged in lint

Reply:
It is, just add a `Suppress` annotation